docs/src/main/sphinx/sql/create-catalog.md

```diff
@@ -0,0 +1,44 @@
+# CREATE CATALOG
+
+## Synopsis
+
+```text
+CREATE CATALOG
+catalog_name
+USING connector_name
+[ WITH ( property_name = expression [, ...] ) ]
+```
+
+## Description
+
+Create a new catalog using the specified connector. The catalog is ready to be
+used as soon as this query completes.
+
+The optional `WITH` clause can be used to set properties
+on the newly created catalog. Property names can be double quoted, which is
+required if they contain special characters, like `-`.
+
+:::{warning}
+The complete CREATE CATALOG query can be logged, and visible in the {doc}`Web
+</admin/web-interface>`. This includes any sensitive properties, like passwords
+and other credentials. See {doc}`secrets </security/secrets>`.
+:::
+
+:::{note}
+This command requires [catalog management type](/admin/properties-catalog) to
+be set to `dynamic`.
+:::
+
+## Examples
+
+Create a new catalog called `postgres` using the PostgreSQL conenctor:
+
+```sql
+CREATE CATALOG postgres USING postgresql
+WITH ("connection-url" = 'jdbc:pg:localhost:5432');
```

Comment:
Expand to use env secrets with username and password and maybe some other postgresql connector config. And explain